### PR TITLE
Update to MkDocs Material v6

### PR DESCRIPTION
**Pull request by RDIL**

### CODE REVIEW

**Comment by fkorotkov** [comment]
Have you tried running it? The top navigation seems missing. Seems `theme/partials` is not picked up.

**Comment by RDIL** [comment]
Thats odd, it worked just fine for https://docs.rdil.rocks.  I didn't try on this site, but could it possibly be that it uses a custom home page?

**Comment by fkorotkov** [comment]
Needs an investigation. Probably the `nav-item` partial should be also synced with upstream.

**Comment by RDIL** [comment]
@fkorotkov I fixed it, it should now work, as it did for me locally.

File: .cirrus.yml [hunk 1]
```diff
@@ -1,9 +1,9 @@
 container:
   cpu: 1
-  memory: 1G
+  memory: 2G
```
Comment (RDIL) [inline .cirrus.yml:3]: When testing locally I ran out of memory so this just bumps it a tiny bit.  If need be, I can change it back.

**Review by fkorotkov — APPROVED** [review]

**Comment by fkorotkov** [comment]
Awesome! Thank you for working on it!

**Pull request merged**